src/components/Pages/Homepage/Products/Products.tsx

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import styles from './Products.module.css';
+import zeroTrustSvg from '../../../Icon/teleport-svg/zero-trust.svg';
+import MWISvg from '../../../Icon/teleport-svg/mwi.svg';
+import IGSvg from '../../../Icon/teleport-svg/identity-governance.svg';
+import ISSvg from '../../../Icon/teleport-svg/identity-security.svg';
+
+interface ProductFeature {
+  title: string;
+  description: string;
+  href?: string;
+}
+
+interface ProductCategory {
+  id: string;
+  title: string;
+  description: string;
+  iconColor?: string;
+  iconComponent: any;
+  features: ProductFeature[];
+}
+
+interface ProductsProps {
+  className?: string;
+}
+
+const productCategories: ProductCategory[] = [
+  {
+    id: 'zero-trust-access',
+    title: 'Zero Trust Access',
+    description: 'Easy access to all your infrastructure on a foundation of cryptographic identity',
+    iconColor: '#EEEAFA',
+    iconComponent: zeroTrustSvg,
+    features: [
+      {
+        title: 'Enroll and Protect Your Infrastructure',
+        description: 'Apps, servers, databases, Kubernetes, desktops, & more',
+        href: '/enroll-resources/'
+      },
+      {
+        title: 'VNet: Build without VPNs',
+        description: 'Secure app & SSH access with no VPNs or proxies',
+        href: '/enroll-resources/application-access/guides/vnet/'
+      },
+      {
+        title: 'Secure MCP (Protect the Vibes)',
+        description: 'Secure MCP integration with granular audit trail',
+        href: 'https://goteleport.com/use-cases/secure-model-context-protocol/'
```

Comment:
This is fine for now; we'll eventually want to update this with the appropriate docs page.